fmriprep/workflows/sbref.py

```diff
@@ -47,7 +47,7 @@ def sbref_preprocess(name='SBrefPreprocessing', settings=None):
 
     mean = pe.Node(fsl.MeanImage(dimension='T'), name='SBRefMean')
     inu = pe.Node(ants.N4BiasFieldCorrection(dimension=3), name='SBRefBias')
-    bet = pe.Node(BETRPT(frac=0.6, mask=True), name='SBRefBET')
+    bet = pe.Node(BETRPT(generate_report=True, frac=0.6, mask=True), name='SBRefBET')
```

Comment:
Since these interfaces are "report capable", there is no need for this ```generate_report``` right? That should be ```True``` always, WDYT?